py/desispec/tile_qa.py

```diff
@@ -51,7 +51,7 @@ def compute_tile_qa(night, tileid, specprod_dir, exposure_qa_dir=None, group='cu
         log.error(f"no spectra files in {tiledir}")
         return None, None
 
-    fmap=read_fibermap(spectra_files[0])
+    fmap = vstack([read_fibermap(spectra_file) for spectra_file in spectra_files])
```

Comment:
Since the spectra files are big and now gzipped, reading the fibermap from them can be slow and sensitive to current I/O performance.  Consider reading from the smaller not-gzipped coadd files instead (FIBERMAP for quantities that are independent of exposure like targeting info, or EXP_FIBERMAP for the per-exposure rows).

Reply:
Thank for the suggestion.

Actually, with looking at the code, I realize that the coadds extensions are read + stacked further below in the code:
https://github.com/desihub/desispec/blob/7f27030dca6c2d68cea16a67cd64b993ea68d342/py/desispec/tile_qa.py#L130-L178

So I think we can just get read of those lines reading the spectra files (they are just here to get the list of expids), move the lines I highlighted (reading+stacking coadds) upper in the code, and get the list of expids from the  exp_fibermaps table.

I ll try/test that later today.

Reply:
The commit https://github.com/desihub/desispec/pull/1767/commits/cd405b4701b7679e1ee79461d7d8370268dd6196 makes the changes I mentioned.